CIP-0014/CIP-0014.md

```diff
@@ -0,0 +1,169 @@
+---
+CIP: 14
+Title: User-Facing Asset Fingerprint 
+Authors: Matthias Benkort <matthias.benkort@iohk.io>, Rodney Lorrimar <rodney.lorrimar@iohk.io>
+Comments-Summary: A common format for displaying a user-facing identifier for assets.
+Comments-URI: https://github.com/cardano-foundation/CIPs/wiki/Comments:CIP-0014
+Status: Draft
+Type: Standards
+Created: 2020-02-01
+License: CC-BY-4.0
+---
+
+# Abstract
+
+This specification defines a user-facing asset fingerprint as a bech32-encoded blake2b-160 digest of the concatenation of the policy id and the asset name.
+
+# Motivation
+
+The Mary era of Cardano introduces the support for native assets. On the blockchain, native assets are uniquely identified by both their so-called policy id and asset name. Neither the policy id nor the asset name are intended to be human-readable data. 
+
+On the one hand, the policy id is a hash digest of either a monetary script or a Plutus script. On the other hand, the asset name is an arbitrary bytestring of up to 32 bytes (which does not necessarily decode to a valid UTF-8 sequence). In addition, it is possible for an asset to have an empty asset name, or, for assets to have identical asset names under different policies. 
+
+Because assets are manipulated in several user-facing features on desktop and via hardware applications, it is useful to come up with a short(er) and human-readable identifier for assets that user can recognize and refer to when talking about assets. We call such an identifier an _asset fingerprint_.
+
+# Specification
+
+We define the asset fingerprint in pseudo-code as:
+
+```
+assetFingerprint := encodeBech32
```

Comment:
do we want to use bech32 or the new bech32m to address this issue: https://github.com/sipa/bech32/issues/51

Reply:
Answer: best to keep everything on standard bech32 to avoid confusion